subxt/src/utils/mod.rs

```diff
@@ -18,6 +18,9 @@ use derivative::Derivative;
 
 pub use account_id::AccountId32;
 pub use era::Era;
+
+#[cfg(feature = "jsonrpsee")]
```

Comment:
You need `#[cfg(all(feature = "jsonrpsee", feature = "native"))]` for this to compile.

It might be better to just import `url::Url` directly to get rid of the conditional compilation here....

Reply:
Yup; that or just check that the string starts with `https://` or `wss://`